src/constant/error.ts

```diff
@@ -4,12 +4,12 @@ export const ERRORS = {
   MISSING_APP_NAME: 'Missing app name',
   PACKAGE_JSON_NOT_FOUND: 'package.json not found',
   PROJECT_FOLDER_EXISTS:
-    'Project folder already exists! Please, run the cli again and choose another project name.',
+    'Project folder already exists! Please run the cli again and choose another project name.',
   VALID_DOMAIN: 'Please enter a valid domain',
   APP_NAME_MISSING: 'App name missing',
-  USER_NAME_EMPTY: "Username can't be empty.",
-  PASSWORD_EMPTY: "Password can't be empty",
-  APP_ID_EMPTY: "App ID can't be empty.",
+  USER_NAME_EMPTY: `Username can't be empty.`,
+  PASSWORD_EMPTY: `Password can't be empty`,
+  APP_ID_EMPTY: `App ID can't be empty.`,
   APP_ID_NUMBER: 'App ID must be a number.',
-  PROXY_EMPTY: "Proxy URL can't be empty."
+  PROXY_EMPTY: `Proxy URL can't be empty.`
```

Comment:
Is it ok with double-quotation for other constants because you didn't wrap it?

Reply:
I converted to double-quotation:
https://github.com/kintone-labs/kintone-cli/pull/106/commits/2a93c4c7bcd7715f09db562ffdaab2fcbf1a9284

Reply:
Sorry, I mean that there is some use `` and some use ''.
I don't understand the difference, so please tell me if you have any reasons.
And in the source code, I think we don't use double quotation and unify with single quotation, so please use it.